File: lua/refactoring/tests/refactor/106/c/with-comments/extract.start.c
#include <stdio.h>

/*
 * This is a comment
 * comments are fun
 * wow!
 */
void simple_function(int a) {
  int test = 1, test_other = 1;

  for (int idx = test - 1; idx < test_other; idx++) {
    printf("%d %d", idx, a);
  }
}

File: lua/refactoring/tests/refactor/106/go/with-comment-above/extract.start.go
package main

import "fmt"

/* This is a multi line comment
comments are fun
I like comments as much as I like coconut oil */
func simple_function(a int) {
    var test int = 1
    test_other := 1
    for idx := test - 1; idx < test_other; idx++ {
        fmt.Println(idx, a)
    }
}
